house/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('', views.dashboard, name='dashboard'),
+]
```

Comment:
Why do you need the urls.py here? you can use the views in Xpense/urls.py like that: 
  ```
from house.views import dashboard  
path('house/', dashboard, name="dashboard")
```